docs/api/commands/getallcookies.mdx

```diff
@@ -80,8 +78,7 @@ cy.getAllCookies()
 
 ### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
 
-- `cy.getAllCookies()` will only run assertions you have chained once, and will
-  not [retry](/guides/core-concepts/retry-ability).
+- `cy.getAllCookies()` has no implicit assertions.
```

Comment:
```suggestion
- `cy.getAllCookies()` has no implicit assertions.
- `cy.getAllLocalStorage()`
  [only reads the data once](/guides/core-concepts/retry-ability#Only-queries-are-retried), so any assertions chaining the command will not retry because the yielded value is static.
```